index.html

```diff
@@ -1,10 +1,58 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="mdl-js">
 <head>
     <meta charset="UTF-8">
-    <title>Title</title>
+    <title>Registration</title>
+    <link rel="stylesheet" href="visual.css">
+    <link rel="shortcut icon" href="src/short_icon.png" type="image/png">
 </head>
+
 <body>
-<h1>Day is gone!</h1>
+<<<<<<< HEAD
+<img class="displayed" src="src/short_icon.png" width="90px"/>
+=======
+<img class="displayed" src="src/short_icon.png" />
```

Comment:
This is conflict. replace with:
`<img class="displayed" src="src/short_icon.png" width="90px"/>`